### ISSUE TITLE
Other repos under dnnsoftware do not have the same issue templates etc.

### ISSUE THREAD

**Issue by OllyHodgson** [dnnsoftware/Dnn.Platform#2191]
## Description 

The [PersonaBar](https://github.com/dnnsoftware/Dnn.AdminExperience.Library), [Personabar Extensions](https://github.com/dnnsoftware/Dnn.AdminExperience.Extensions) and [Edit Bar](https://github.com/dnnsoftware/Dnn.EditBar) repositories do not have the same issue templates (etc) as this one. 

Should they all be lined up (so all have the same templates etc), or should issues be handled centrally (e.g. right here)?

## Steps to reproduce

Visit https://github.com/dnnsoftware/Dnn.AdminExperience.Extensions and raise a new issue.

## Current result

Get a blank issue box.

## Expected result

Either:

1) Ensure all of the repos have the same templates and contributor guidelines as this one 

*or*

2) Disable issues on those repos and handle them all here, updating the contribution documentation in those repos accordingly

**Comment by mitchelsellers** [comment]
We have a pending task underway that is going to simplify the repository structure.  Once that is done we will standardize the issue templates across the platform resolving this issue.

Thanks for the suggestion, I'll leave this issue open until the other items have been addressed.

**Comment by sleupold** [comment]
this should have been fixed meanwhile, though I noticed duplicate fields in some of the issue templates.

**Comment by valadas** [comment]
So at this time it looks like they need to be slightly different due to the versions not being the same on each repo, so we need to manually handle those separately. Closing this issue.
